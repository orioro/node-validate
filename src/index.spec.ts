import * as api from './index'

test('public api', () => {
  expect(Object.keys(api).sort()).toMatchSnapshot()
})
