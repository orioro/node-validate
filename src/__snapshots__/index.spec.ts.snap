// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`public api 1`] = `
Array [
  "ERR_VALIDATION",
  "ValidationError",
  "allowValues",
  "normalizeValidationResult",
  "parallelCases",
  "prepareValidate",
  "prohibitValues",
  "sequentialCases",
  "validateAsync",
  "validateAsyncThrow",
  "validateSync",
  "validateSyncThrow",
]
`;
